cypress/fixtures/mockPostDetails.json

```diff
@@ -0,0 +1,59 @@
+{
+  "data": {
+    "id": 1,
+    "user": {
+      "id": 1,
+      "username": "BeeKeeper1"
+    },
+    "title": "New research on waxworms",
+    "description": "Colony collapse disorder of honey bees, Apis mellifera L., is a global problem with no conclusive cause yet accepted. A previous U.S. Army study identified a DNA virus, invertebrate iridescent virus (IIV-6), and two microsporian pathogens, Nosema apis Zander 1909 and Nosema ceranae Fries et al. 1996, in bee samples from a collapsing hive. A PCR-based study using limited samples failed to confirm IIV-6 in collapsing colonies, causing the finding to be questioned. Here we demonstrate that honey bees are very susceptible to the virus. We observed viral inclusion bodies in the cytoplasm of honey bee brood previously inoculated with IIV-6. Electron microscopy revealed massive numbers of viral particles in cells at 3 days post-inoculation. Viral factories and paracrystalline arrays of particles ~125 nm in diameter were observed. Few cells were spared infection, indicating that honey bee larvae are very susceptible to IIV-6 and most tissues are infected.",
+    "image": "https://user-images.githubusercontent.com/79113236/131058964-ba612108-33e3-4bbd-b60e-857aa1a4abb5.png",
+    "created_at": "2021-08-26 14:28:22 -0600",
+    "upvotes": 5,
+    "downvotes": 1,
+    "comments": [
+      {
+        "type": "comment",
+        "id": "1",
+        "attributes": {
+          "user_id": 3,
+          "username": "BeeGuy20",
+          "avatar": "https://tinyurl.com/nj76mca3",
+          "content": "Great research, what can I do as a beekeeper?",
+          "timestamp": "2021-08-26 15:28:22 -0600",
+          "upvotes": 1,
+          "downvotes": 0,
+          "post_id": 1
+        }
+      },
+      {
+        "type": "comment",
+        "id": "1",
+        "attributes": {
```

Comment:
Yeah this is actually totally different with graphQL, there is no "attributes" section and I believe "user" is an object with values.